examples/alerts/alerts.yaml

```diff
@@ -569,17 +569,6 @@ groups:
       severity: critical
 - name: thanos-component-absent
   rules:
-  - alert: ThanosBucketReplicateIsDown
```

Comment:
I think the circleCI test failure is related now, as `alerts.yaml` was changed, so the `thanos-component-absent` group now has 6 alerts instead of 7. I think removing one `someAlert` in [`rules_test.go`](https://github.com/thanos-io/thanos/blob/main/pkg/rules/rules_test.go#L53) should do the trick! 🙂

Reply:
Exactly this. Maybe using these alerts in ruler tests was not a really good idea. These sort of changes shouldn't break something in go tests. IMHO we should clone some of these file on a `testdata` folder in ruler tests and use those instead of alerts.yaml

Reply:
makes sense, changed it!